packages/edge-gateway/src/gateway.js

```diff
@@ -105,10 +103,16 @@ export async function gatewayGet (request, env, ctx) {
     const resourceCid = decodeURIComponent(
       winnerGwResponse.headers.get('etag') || ''
     )
-    const denyListResource = await getFromDenyList(env.DENYLIST, resourceCid)
-    if (denyListResource) {
-      const { status, reason } = JSON.parse(denyListResource)
-      return new Response(reason || '', { status: status || 410 })
+
+    const cidVerificationResponse = await env.CID_VERIFIER.fetch(`${env.CID_VERIFIER_URL}/verification?cid=${resourceCid}`)
```

Comment:
Please rename this to `cidResourceVerificationResponse` for better readability that this is resource cid from etag